website/src/components/subscriptions/create/SubscriptionsCreate.tsx

```diff
@@ -39,13 +40,22 @@ export function SubscriptionsCreateInner({
     const createSubscription = useMutation({
         mutationFn: () => getBackendServiceForClientside().postSubscription(getSubscription(), userId),
         onSuccess: () => {
-            window.location.href = '/subscriptions';
+            toast.success('Successfully created subscription', {
+                position: 'bottom-left',
+                autoClose: 4000,
+                onClose: () => {
+                    window.location.href = '/subscriptions';
+                },
```

Comment:
I think users would find this weird. Either don't open a new page or do it immediately...

If we go to a new page, we could also show the toast there.

Reply:
I did not want to pass state to the other page because of complexity. 
Maybe we stay on the same page, but offer two new buttons, one to create a new subscription and one to go back to the overview?

Reply:
As discussed: For now we only show the success on the button, and have a dedicated design including the "edit subscription" page.